Updated Statamic docs (#121)
rapidez/docs: src/5.x/packages/statamic.md

The Rapidez with [Statamic](https://statamic.com/) integration! 🤝🚀

::: info Version 8
This documentation is for [`rapidez/statamic`](https://github.com/rapidez/statamic) `^8.0`, which requires PHP 8.4 and [Statamic 6](https://statamic.dev/).
:::

[[toc]]

## Features

- Products, categories, and read-only product attributes (plus attribute options) are integrated through [Runway](https://github.com/statamic-rad-pack/runway)

> - Brand landing pages use Statamic's **`brands` collection** (page builder and SEO), still tied to Magento brand options for listings and filters; the Runway brands resource is hidden in the control panel so you edit brands in one place

Reviewer: The wording of this point here feels confusing. I know what it means because I know all of the context surrounding the brands collection, but if you have no idea then this doesn't make any sense.

I'd suggest something like:

```suggestion
- Brand landing pages use a regular Statamic collection named `brands` (with page builder and SEO) that is linked to the Runway resource, which is unlike other Runway resources. The Runway resource itself is hidden.
```

## Static caching

> Statamic comes with [static caching](https://statamic.dev/static-caching) and with this package we're adding the middleware that handles that from Statamic to all Rapidez web routes. When you configure static caching with Statamic it will also be applied to all Rapidez routes!

Reviewer: ```suggestion
Statamic comes with [static caching](https://statamic.dev/static-caching) and this package adds the middleware for that. This means that when you configure static caching with Statamic, it will also be applied to all Rapidez routes.
```
